### PR TITLE
OTWO-5465 - updates to License Obligation tables migrations

### PR DESCRIPTION
**Pull request by msk999**

### CODE REVIEW

File: test/factories/license_license_permissions.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    #   status LicensePermissionStatus.statuses['permitted']
+    # end
+
+    # factory :forbidden_license_permission do
+    #   status LicensePermissionStatus.statuses['forbidden']
+    # end
+
+    # factory :required_license_permission do
+    #   status LicensePermissionStatus.statuses['required']
+    # end
```
Comment (bdsdeployer) [inline test/factories/license_license_permissions.rb]: Do we not need these commented out special factories?

File: app/assets/stylesheets/admin/custom.css [hunk 2]
```diff
@@ -3,3 +3,68 @@
...
+    box-shadow: 0 1px 1px rgba(0,0,0,0.1), 0 1px 0 0 rgba(255,255,255,0.8) inset;
+    border: solid 1px #c7c7c7;
+    border-color: #c7c7c7;
+    border-top-color: #d3d3d3;
+    border-right-color: #c7c7c7;
+    border-bottom-color: #c2c2c2;
+    border-left-color: #c7c7c7;
+    text-shadow: #fff 0 1px 0;
+    color: #5E6469;
+}
```
Comment (bdsdeployer) [inline app/assets/stylesheets/admin/custom.css]: There seems to be a lot of CSS formatting -- what there nothing for the current admin pages that worked? Isn't this just Bootstrap?

**Comment by PDegenPortnoy** [comment]
Sorry, I was logged into the bdsdeployer account when I did the review.

File: app/assets/stylesheets/admin/custom.css (hunk 2 above)
Comment (msk999) [reply app/assets/stylesheets/admin/custom.css]: I took this from the current admin pages.  No particular reason other than it looks good.

File: test/factories/license_license_permissions.rb (hunk 1 above)
Comment (msk999) [reply test/factories/license_license_permissions.rb]: I'll remove the commented lines

File: test/controllers/oh_admin/license_permissions_controller_test.rb [hunk 3]
```diff
@@ -0,0 +1,27 @@
+require 'test_helper'
+
+describe 'OhAdmin::LicensePermissionsController' do
+  let(:admin) { create(:admin) }
+
+  let(:license_permissions) { create(:license_permission, license_right: create(:license_right, name: 'Permitted')) }
+  let(:license) { create(:license) }
+
+  before do
+    login_as admin
```
[the remark this answers is not in the document]
Comment (msk999) [reply test/controllers/oh_admin/license_permissions_controller_test.rb]: ok

File: app/assets/stylesheets/admin/custom.css (hunk 2 above)
Comment (PDegenPortnoy) [reply app/assets/stylesheets/admin/custom.css]: Is it possible to reuse the existing CSS?

**Comment by msk999** [comment]
retest this please

File: app/assets/stylesheets/admin/custom.css (hunk 2 above)
Comment (msk999) [reply app/assets/stylesheets/admin/custom.css]: I have not been able to use the active_admin stylesheets in the form.  That's why I copied what I needed into custom.css

Comment (PDegenPortnoy) [reply app/assets/stylesheets/admin/custom.css]: Understood.

**Pull request merged**